app/livechat/server/hooks/beforeDelegateAgent.js

```diff
@@ -3,9 +3,7 @@ import { callbacks } from '../../../callbacks';
 import { settings } from '../../../settings';
 import { Users, LivechatDepartmentAgents } from '../../../models';
 
-callbacks.add('livechat.beforeDelegateAgent', (options = {}) => {
-	const { department, agent } = options;
-
+callbacks.add('livechat.beforeDelegateAgent', (agent, department) => {
```

Comment:
I think it's best to keep using a key value object, if you need another different item, it won't break everywhere. This change actually doenst fix or solves anything
```suggestion
callbacks.add('livechat.beforeDelegateAgent', (agent, { department }) => {
```